python_modules/libraries/dagster-gcp/dagster_gcp/gcs/sensor.py

```diff
@@ -0,0 +1,34 @@
+import dagster._check as check
+from google.cloud import storage
+
+MAX_KEYS = 1000
+
+
+def get_gcs_keys(bucket, prefix="", since_key=None, gcs_session=None):
+    """Credits to @ajohnson5 for the original implementation of this method."""
+    check.str_param(bucket, "bucket")
+    check.str_param(prefix, "prefix")
+    check.opt_str_param(since_key, "since_key")
+
+    if not gcs_session:
+        gcs_session = storage.Client()
+
+    contents = list(
+        gcs_session.list_blobs(
+            bucket_or_name=bucket,
+            delimiter="",
+            page_size=MAX_KEYS,
+            prefix=prefix,
+        )
```

Comment:
Should we provide the `start_offset` here, similarly to what we do with `StartAfter` in `get_s3_keys`? See [here](https://github.com/dagster-io/dagster/blob/fc2869650894861b3bec1c541eb4e3a87ce167d2/python_modules/libraries/dagster-aws/dagster_aws/s3/sensor.py#L24)